pytest.ini

```diff
@@ -1,8 +1,13 @@
 [pytest]
 
+# Don't run the tests marked as slow by default.
+addopts = -m 'not slow'
```

Comment:
How exactly does this interact with options provided on the command line such as `-m slow` or `-m not XFAIL` or something?

Reply:
There are multiple ways to run the tests:

#### Using `pytest` from the command line
This is the only place where this `addopts` makes a difference. It composes nicely with other arguments passed with the `-m` flag. So running `pytest -m xfail` behaves the same as running `pytest -m "not slow and xfail"`. Slow tests can still be run using `pytest -m slow` and all tests can be run using `pytest -m "slow or not slow"`.

#### Using `sympy.test()`
This (hopefully) maintains the same interface as before. There is a `slow` kwarg that defaults to `False` and automatically adds the `-m "not slow"` to the pytest call. Calling `sympy.test(slow=True)` is equivalent to `pytest -m slow` on the command line. There is no option to run all tests (slow and not slow) via this interface, but AFAIK this is the same as before.

#### Using `bin/test`
This eventually calls `sympy.test()` so is the same behaviour as above.